docs/tutorial/part-one/index.md

```diff
@@ -53,7 +53,7 @@ Gatsby has a built-in development server. Let's start it up.
 
 ```sh
 cd tutorial-part-one
-npm run develop
+gatsby develop
```

Comment:
i think the original command was correct here

Reply:
They're both correct — David made the point in some offline communication that we're not consistent here about the "right" way to start the dev server. I agree that "gatsby develop" is generally the "right" way to start the gatsby dev server even if `npm run develop` works perfectly well too.